app/src/main/res/layout/auth_url_container.xml

```diff
@@ -44,6 +44,7 @@
                 android:hint="@string/other_url"
                 android:inputType="textUri"
                 android:imeOptions="actionGo"
+                app:validateType='@{"url"}'
```

Comment:
Check if adding `validateType` makes adding this URL mandatory.

Reply:
Yes, that's the problem currently. Hence, I had added WIP in the title after I realized this.

Reply:
Please review now.